ckanext/canada/plugin/public_plugin.py

```diff
@@ -212,6 +215,56 @@ def column_types(self, existing_types: Dict[str, Type[ColumnType]]):
     def get_blueprint(self) -> List[Blueprint]:
         return [canada_views]
 
+    def _canada_citation_map(self, cite_data: DataDict,
+                             pkg_dict: DataDict) -> Dict[str, Any]:
+        lang = 'en'
+        if has_request_context():
+            lang = h.lang()
+
+        if (
+          pkg_dict.get('org_section') and
+          lang in pkg_dict['org_section']
+          and pkg_dict['org_section'][lang]):
+            cite_data['publisher'] += ' - ' + pkg_dict['org_section'][lang]
```

Comment:
IIRC the suggested style for indenting if conditions is to add an extra (4-space) indent. Also this could be simplified to:

```suggestion
        if pkg_dict.get('org_section', {}).get(lang):
            cite_data['publisher'] += ' - ' + pkg_dict['org_section'][lang]
```